internal/user/model/user_membership_view.go

```diff
@@ -0,0 +1,87 @@
+package model
+
+import (
+	"time"
+
+	"github.com/caos/zitadel/internal/model"
+)
+
+type UserMembershipView struct {
+	UserID      string
+	MemberType  MemberType
+	AggregateID string
+	ObjectID    string
```

Comment:
What's the difference between `AggregateID` and `ObjectID`?

Reply:
for org and project nothing, but on project grant we have a aggregate id and a grant id

Reply:
So i would add a comment 
```suggestion
	AggregateID string
	//ObjectID optional field for sub-objects in aggregates
	ObjectID    string
```